### DISCUSSION TITLE
[Script request] Docker VM

### DISCUSSION THREAD

**Discussion by muescha** [community-scripts/ProxmoxVE#148] [Request script]
### Application Name

Docker

### Website

https://github.com/docker

### Description

I see at https://pve.proxmox.com/wiki/Linux_Container this:

> If you want to run application containers, for example, Docker images, it is recommended that you run them inside a Proxmox QEMU VM. This will give you all the advantages of application containerization, while also providing the benefits that VMs offer, such as strong isolation from the host and the ability to live-migrate, which otherwise isn’t possible with containers.

It would be nice to have also a script for `Docker VM`

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Mellowlynx** [comment]
Why not just deploy a Debian or Ubuntu VM and install docker on there?
One VM script can be used for thousands of things, instead of making thousands scrips for all just one thing.

**Reply by muescha** [reply]
Home Assistant is also available as a VM script. Since new users often rely on this script library, one use case is that they may want to install Docker as a VM. Installing Docker as a VM is particularly recommended in this scenario.

**Comment by muescha** [comment]
marked as solved but no Docker VM visible?
